sdk/storage/storage-queue/src/QueueClient.ts

```diff
@@ -1103,22 +1102,20 @@ export class QueueClient extends StorageClient {
       //  URL may look like the following
       // "https://myaccount.queue.core.windows.net/myqueue?sasString".
       // "https://myaccount.queue.core.windows.net/myqueue".
-      // or an emulator URL that starts with the endpoint `http://127.0.0.1:10001/devstoreaccount1`
+      // IP style Endpoints - `http://127.0.0.1:10001/devstoreaccount1`
```

Comment:
minor: would better align the examples with existing ones, e.g. "[http://127.0.0.1:10001/devstoreaccount1/myqueue](http://127.0.0.1:10001/devstoreaccount1/myqueue)"

Reply:
I think we should replace "IP style endpoints" with standard terminology: `IPv4 address hosts` or similar?

Reply:
Updated both!